app/advanced-debug.tsx

```
@@ -1,27 +1,37 @@
-import { useFeatureFlags } from '@/hooks/useFeatureFlags';
+import { invertAgentMode, useFeatureFlags } from '@/hooks/useFeatureFlags';
 import { useAppState } from '@/hooks/useAppState';
 
 export default function AdvancedDebug() {
-    const { agentMode, overrideFeatureFlag } = useFeatureFlags();
+    const { agentMode, isAgentSwitched, overrideFeatureFlag } = useFeatureFlags();
     const { setRequireNewChat } = useAppState();
 
     function toggleAgentMode() {
-        const newMode = agentMode === 'introvert' ? 'extrovert' : 'introvert';
+        const newMode = invertAgentMode(agentMode!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)